host/sgx/sgxquote.c

```diff
@@ -123,8 +132,17 @@ static quote3_error_t (*_sgx_qv_verify_quote)(
 
 #define UNLOAD_SGX_DCAP_LIB(module) dlclose(module)
 
+// Before: the in-process call path is used unless the environment variable
+// SGX_AESM_ADDR is set. After: Regardless of the existence and value of
+// SGX_AESM_ADDR, out-of-process call path is the default setting.
 #define SGX_DCAP_IN_PROCESS_QUOTING() (getenv("SGX_AESM_ADDR") == NULL)
```

Comment:
Can we deprecate "SGX_AESM_ADDR" and use only "SGX_USE_IN_PROCESS_QUOTING" going forward?

"SGX_USE_IN_PROCESS_QUOTING" can be 0 or 1 (non-zero). If it's not set, it will be 1 in Windows and 0 in Linux.

Reply:
That sounds good to me too. I can add some comments for it in the file.